acceptance/acceptance_test.go

```diff
@@ -66,6 +68,7 @@ func init() {
 	flag.BoolVar(&LogRequests, "logrequests", false, "Log request and responses from testserver")
 	flag.BoolVar(&LogConfig, "logconfig", false, "Log merged for each test case")
 	flag.BoolVar(&SkipLocal, "skiplocal", false, "Skip tests that are enabled to run on Local")
+	flag.BoolVar(&Dbr, "dbr", false, "The tests are running on DBR.")
```

Comment:
If flag is called "-dbr" I assume I can run "go test -dbr" and this will do everything on DBR.

The name of the flag and description should be more specific, reflecting what test runner is actually going to do. in this case, it will use /Workspace to create temporary directory, so it should just say that:

-workspace-tmpdir "Create temp directory in /Workspace/Users/<username>/acceptance (for DBR testing)"

Reply:
After reviewing the rest - perhaps even better approach is to add -tmproot flag here and calculate tmpdir in the runner script.